tools/clang/tools/dxclib/dxc.cpp

```diff
@@ -41,6 +41,7 @@
 #include "dxc/Support/Unicode.h"
 #include "dxc/Support/WinIncludes.h"
 #include "dxc/Support/WinFunctions.h"
+#include "dxc/dxcerrors.h"
```

Comment:
I think the simplest solution to the linux build failure is to move this include to the `#ifdef _WIN32` conditional. The catching code is already win32 only